Plugins/CTFAK.Decompiler/FTDecompile.cs

```diff
@@ -200,16 +200,19 @@ public void Execute(IFileReader reader)
                 var key = game.frameitems.Keys.ToArray()[i];
                 var item = game.frameitems[key];
                 var newItem = TranslateObject(mfa,game,item);
+                //Object Section
                 if (newItem.Loader == null)
                 {
-                    throw new NotImplementedException("Unsupported Object: "+newItem.ObjectType);
+                    throw new NotImplementedException("Fucked Object: "+newItem.ObjectType);
```

Comment:
Please revert back to normal error message

Reply:
ok